doc/README.md

```diff
@@ -53,3 +53,48 @@ You can also check out the
 [awesome flame repository](https://github.com/flame-engine/awesome-flame#articles--tutorials),
 It contains quite a lot of good tutorials and articles written by the community to get you started
 with Flame.
+
+```{toctree}
```

Comment:
This breaks it on github view: 
![image](https://user-images.githubusercontent.com/882703/135310555-3d50cc8c-a924-4956-99dd-a523e419eac3.png)

I wonder if we can find a way to make it work both ways. or at least omit it on github

Reply:
I've put the table of contents into a separate file. This way the readme can stay exactly the way it is now, but there is also a new `index.md` file, and new `toc.rst` file.